bip-0003.md

```diff
@@ -527,6 +527,7 @@ mentioned in the [Changelog](#changelog) section.
   - A BIP in Draft status may be set to Closed by anyone if it appears to have stopped making progress for at least a
```

Comment:
For a follow-up maybe

s/A BIP in Draft status may be set to Closed by anyone/Anyone may propose that a BIP be updated to Closed status/